local_flow/intent/src/utils.py

```diff
@@ -17,3 +20,60 @@ def return_json_file_content(file_name: str):
         data = json.load(json_file)
 
     return data
+
+class ExperimentTracker:
```

Comment:
Could we add docstrings to this class and its associated methods? This will help users understand their purpose and usage. Also it will follow the documentation patterns implicitly defined in the project

Reply:
Done 👍  let me know if there are any further changes.

Reply:
Nice docs